src/jquery-sticky-header-footer.js

```diff
@@ -48,7 +48,8 @@
                 @method tearDown
              */
             tearDown: function() {
-                var element = this.first();
```

Comment:
`this.first()` was throwing here, I assume because it was looking for a jQuery object? Not sure.